modules/@apostrophecms/ui/ui/apos/components/AposCellType.vue

```diff
@@ -15,7 +15,7 @@ export default {
   mixins: [ AposCellMixin ],
   methods: {
     label(typeName) {
-      return (apos.modules[typeName] && apos.modules[typeName].label) || typeName;
+      return this.$t((apos.modules[typeName] && apos.modules[typeName].label) || typeName);
```

Comment:
this seems inconsistent with the more common pattern of localizing in the template block. the button, label, and context menu item components here use that, for a few.